test/core/autoscaling/system_status.test.ts

```diff
@@ -168,17 +168,17 @@ describe('SystemStatus', () => {
         });
 
         // @ts-expect-error Overwriting readonly private prop
-        systemStatus.currentHistorySecs = 5;
+        systemStatus.currentHistoryMillis = 5_000;
```

Comment:
i think these tests were passing because we changed the value to just `5` instead of `5000`